Integrations/integration-C2sec.yml

```diff
@@ -0,0 +1,236 @@
+         return  {
+                    'Type': entryTypes['note'],
+                    'ContentsFormat': formats['json'],
+                    'Contents': resp,
+                    'HumanReadable': md,
+                    'EntryContext': {'c2secScanQueryStatus': resp}
+                }
+       else:
+           return ('Error querying a workitem [%d] - %s' % (call.status_code, call.text))
+
+    def queryIssues(domain):
+       params = {'apikey': demisto.params()['apikey'], 'domain': domain }
+       headers = {'Content-Type': 'application/json', 'Accept': 'application/json'}
+       call = requests.get(demisto.params()['endpointURL'] + 'iRiskQueryIssues', headers=headers, params=params)
+       if call.status_code == requests.codes.ok:
+         md = tableToMarkdown('Query Issues Result', call.json()['issueList'])
+         return  {
+                    'Type': entryTypes['note'],
+                    'ContentsFormat': formats['json'],
+                    'Contents': call.json(),
+                    'HumanReadable': md,
+                    'EntryContext': {}
+                }
+       else:
+           return ('Error querying for issues [%d] - %s' % (call.status_code, call.text))
+
+
+    def rescan(domain):
+       params = {'apikey': demisto.params()['apikey'], 'domain': domain}
+       headers = {'Content-Type': 'application/json', 'Accept': 'application/json'}
+       call = requests.get(demisto.params()['endpointURL'] + 'iRiskRescanCompany', headers=headers, params=params)
+       if call.status_code == requests.codes.ok:
+         md = tableToMarkdown('Rescan Results', call.json())
+         return  {
+                    'Type': entryTypes['note'],
+                    'ContentsFormat': formats['json'],
+                    'Contents': call.json(),
+                    'HumanReadable': md,
+                    'EntryContext': {'c2secScanRequest': call.json()}
+                }
+       else:
+           return ('Error rescanning the company  [%d] - %s' % (call.status_code, call.text))
+
+
+    def queryComponent(domain, component):
+       params = {'apikey': demisto.params()['apikey'], 'domain': domain, 'component': component }
+       headers = {'Content-Type': 'application/json', 'Accept': 'application/json'}
+       call = requests.get(demisto.params()['endpointURL'] + 'iRiskQueryComponentData', headers=headers, params=params)
+       if call.status_code == requests.codes.ok:
+         md = tableToMarkdown('Query Component Result', call.json()['data'])
+         return  {
+                    'Type': entryTypes['note'],
+                    'ContentsFormat': formats['json'],
+                    'Contents': call.json(),
+                    'HumanReadable': md,
+                    'EntryContext': {}
+                }
+       else:
+           return ('Error querying for component [%d] - %s' % (call.status_code, call.text))
+
+    def createIncident(c2sec, domain, component):
```

Comment:
You are not using it, maybe you forgot a command?

Reply:
removed